spec/models/importers/number_of_views_by_organisation_spec.rb

```diff
@@ -5,8 +5,9 @@
     let!(:organisation) { create(:organisation, slug: 'the-slug') }
     let!(:content_item_first) { create(:content_item, base_path: 'the-link/first', organisations: [organisation]) }
     let!(:content_item_second) { create(:content_item, base_path: 'the-link/second', organisations: [organisation]) }
+    let!(:content_item_third) { create(:content_item, base_path: 'the-link/third', organisations: [organisation]) }
```

Comment:
Minor thing but not sure you need a third item to test with - I think two items and a batch of `1` followed by a batch of `2` would accomplish the same.